examples/umami/common.rs

```diff
@@ -521,12 +557,15 @@ pub async fn anonymous_contact_form(user: &GooseUser, english: bool) -> GooseTas
                     }
 
                     // Build contact form parameters.
-                    // @TODO: dynamically generate the content.
+                    let name = random_words(2, english).join(" ");
+                    let email = format!("{}@example.com", random_words(1, english).pop().unwrap());
```

Comment:
Could this be invalid and lead to an error?

Reply:
I have not triggered any errors in my testing. It seems validation of the email address is done by this library:
https://github.com/egulias/EmailValidator/blob/master/src/Validation/RFCValidation.php

In any case, even if it _can_ generate an error, an infrequent error condition still seems like a valid test?

Reply:
I let the load test run for an hour and saw no errors:
```
 POST anon /en/contact    |         9,200 |         0 (0%) |     2.56 |    0.00
 POST anon /en/search     |         9,198 |         0 (0%) |     2.56 |    0.00
 POST anon /es/contact    |         3,123 |         0 (0%) |     0.87 |    0.00
 POST anon /es/search     |         3,123 |         0 (0%) |     0.87 |    0.00
```